README-UBUNTU-14.04

```diff
@@ -0,0 +1,14 @@
+Setup steps - non hdfs
+1)cd scribe; rm -f debian; ln -sf debian_nonhdfs debian
+2)sudo vi /etc/apt/sources.list.d/inmobi.list
+add the entry deb http://repo.corp.inmobi.com:9999/precise ./
+3)sudo apt-get update
+4)sudo apt-get install debhelper autotools-dev inmobi-techplatform-libthrift-dev inmobi-techplatform-libfb303 libboost-thread-dev  libstdc++6-4.4-dev libevent-dev inmobi-techplatform-thrift-compiler libboost-dev libboost-test-dev libboost-program-options-dev libboost-system-dev libboost-filesystem-dev libevent-dev automake libtool flex bison pkg-config g++ libssl-dev
+5) creating sym links for pointing the multi threaded libboost to single threaded libboost
```

Comment:
I tried to do some quick search and could find boost multi-threaded libraries for ubuntu 14:

http://packages.ubuntu.com/trusty/libboost-thread1.54-dev
http://packages.ubuntu.com/trusty/libboost-thread1.55-dev
http://stackoverflow.com/questions/26716202/linux-ubuntu-14-04-boost-libs-and-thread-compilations

Scribe is essentially a multi-threaded application. We need to be extremely careful not to impact this aspect.
Also this linking is done for archive files, whereas scribed essentially links with shared libraries.